#!/bin/sh -e
## 40-kolabd-pid-file.dpatch by Noèl Köthe <noel@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This is the patch to correct the pid file path

if [ $# -lt 1 ]; then
echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
      -patch) patch $patch_opts -p1 < $0;;
      -unpatch) patch $patch_opts -p1 -R < $0;;
      *)
      echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
      exit 1;;
esac

exit 0

@DPATCH@
--- kolabd-1.9.4/kolabd.orig	2006-01-25 16:32:11.000000000 +0100
+++ kolabd-1.9.4/kolabd	2006-01-25 16:32:46.000000000 +0100
@@ -52,8 +52,8 @@
 openlog('kolabd', 'cons, pid', 'user');
 
 my $prefix = $Kolab::config{'prefix'};
-my $pidfile = IO::File->new("$prefix/var/kolab/kolab.pid", 'w+')
-    || die "Unable to open PID file `$prefix/var/kolab/kolab.pid'";
+my $pidfile = IO::File->new("/var/run/kolab.pid", 'w+')
+    || die "Unable to open PID file `/var/run/kolab.pid'";
 print $pidfile $$;
 undef $pidfile;
 
